schema/theme.schema.json

```diff
@@ -105,26 +120,140 @@
         "item-color-inverted-selected": {
           "type": "string",
           "title": "Item colour - inverted selected",
-          "description": "Defines the item text/icon colour when selected. Should be a colour that provides good contrast against the Item selected colour",
+          "description": "Defines the item text / icon colour when selected. Should be a colour that provides good contrast against the item selected colour.",
           "default": "",
           "_backboneForms": "ColourPicker"
         },
         "visited": {
           "type": "string",
           "title": "Visited colour",
-          "description": "Defines the item colour when it has been visited",
+          "description": "Defines the item colour when it has been visited.",
           "default": "",
           "_backboneForms": "ColourPicker"
         },
         "visited-inverted": {
           "type": "string",
           "title": "Visited colour - inverted",
```

Comment:
As this variable is only used by presentation components I would change the title for consistency with the other component item colours, _Item colour - inverted visited_.

Reply:
This should now be resolved